ui/component/publishForm/view.jsx

```diff
@@ -387,6 +405,12 @@ function PublishForm(props: Props) {
 
     const isValid = uri && isURIValid(uri);
     if (uri && isValid && checkAvailability && name) {
+      const otherUriToResolve = alternateCapitalizeString(uri);
+      if (otherUriToResolve) {
+        resolveUri(otherUriToResolve);
+        checkAvailability(alternateCapitalizeString(name));
+      }
+
       resolveUri(uri);
       checkAvailability(name);
       updatePublishForm({ uri });
```

Comment:
Don't quite understand the overall change (leave it to Tom), but 
- I think the additional `resolve` is unnecessary?  If not mistaken, we `resolve` just to get the bid for the uri of interest -- so would it matter for this "alternate" thing?